src/Payments/CardPayment.res

```diff
@@ -253,13 +270,32 @@ let make = (
                   )}
                   appearance=config.appearance
                   type_="tel"
-                  className="tracking-widest w-full"
+                  className={`tracking-widest w-full ${innerLayout === Compressed &&
+                      cvcError->String.length > 0
+                      ? "!border-l-0"
+                      : ""}`}
                   maxLength=4
                   inputRef=cvcRef
                   placeholder="123"
                 />
               </div>
             </div>
+            <RenderIf
+              condition={cardError->String.length > 0 ||
+              cvcError->String.length > 0 ||
+              expiryError->String.length > 0}>
+              <div
+                className="Error pt-1"
+                style={ReactDOMStyle.make(
+                  ~color=themeObj.colorDangerText,
+                  ~fontSize=themeObj.fontSizeSm,
+                  ~alignSelf="start",
+                  ~textAlign="left",
+                  (),
+                )}>
+                {React.string("Invalid input")}
```

Comment:
Why is localeString not used here?

Reply:
We don't have as of now. We have to update this with a better UX. Currently is a temp fix.